app/controllers/asset/partnership/PartnershipStartDateController.scala

```diff
@@ -59,7 +61,7 @@ class PartnershipStartDateController @Inject()(
 
   def onSubmit(mode: Mode): Action[AnyContent] = (standardActionSets.verifiedForIdentifier andThen nameAction).async {
     implicit request =>
-
+      val form = formProvider.withConfig(messagePrefix, request.userAnswers.whenTrustSetup)
```

Comment:
I don't believe we need to do it for partnership. Double check with Wendy

Reply:
I don't care about Partnership. We haven't done that one yet. This is just to get the code to compile

Reply:
My worry is that introducing this change means it may linger when we bring partnership in. Wouldn't be too big a change. Could copy the code from reg to make the date optional in the form and then choose the date and message keys depending on whether it's defined or not